### ISSUE TITLE
Incompatibility with Keras 3

### ISSUE THREAD

**Issue by mufernando** [kr-colab/diploSHIC#55]
We need to fix the required version to 2.14

```
Traceback (most recent call last):
  File "/gpfs/projects/kernlab/murillor/micromamba/envs/analysis2/bin/diploSHIC", line 324, in <module>
    from keras.preprocessing.image import ImageDataGenerator
ImportError: cannot import name 'ImageDataGenerator' from 'keras.preprocessing.image' 
```

**Comment by andrewkern** [comment]
@mufernando will just changing this import suffice to make it run with keras 3?

**Comment by mufernando** [comment]
unclear to me rn

https://github.com/keras-team/keras/issues/18467

**Comment by andrewkern** [comment]
this was closed with #56
